### PR TITLE
[Lesson 14] Add new test for empty fields Sign In

### PR DESCRIPTION
**Pull request by TatyanaDovzhich**
Add new test for empty fields Sign In

### CODE REVIEW

File: tests_start_page.py [hunk 1]
```diff
@@ -55,3 +55,46 @@ def test_invalid_sign_in(self):
...
+        self.log.info("SignIn button was click")
+        sleep(1)
+
+        # Verify error
+        error_message = driver.find_element(by=By.XPATH, value=".//div[@class='alert alert-danger text-center']")
+        assert error_message.text == "Error"
+        self.log.info("Error message was verified")
+        sleep(1)
+
+        driver.close()
```
Comment (TatyanaDovzhich) [inline tests_start_page.py:111]: Need to add next test case

**Review by TatyanaDovzhich — COMMENTED** [review]
Looks nice, but need next test case
